main/WebServer.cpp

```diff
@@ -7588,6 +7610,7 @@ namespace http {
 					}
 					root["result"][ii]["idx"] = sd[0];
 					root["result"][ii]["Protected"] = (iProtected != 0);
+					root["result"][ii]["Options"] = sOptions.c_str(); // OTO: output options for all sensors, overriden afterwards depending on sensor type
```

Comment:
Depends on what you want to use the 'options' for, this should not be needed as this is handled by the dummy creation function anyway.
Why is the modification of the options needed? I can not see this ?